client/post-editor/editor-author/index.jsx

```diff
@@ -105,7 +105,9 @@ export default connect(
 
 		const site = getSite( state, siteId );
 		const post = getEditedPost( state, siteId, postId );
-		const author = get( post, 'author', getCurrentUser( state ) );
+		const postAuthor = get( post, 'author' );
+		//default to current user when null or falsey
+		const author = postAuthor ? postAuthor : getCurrentUser( state );
```

Comment:
I trust that this fixed it, but I'm not seeing how. Do you happen to recall the exact call that somehow skipped the default somehow?

Reply:
ah, got it
`_.get( { author: false }, 'author', 'default' )` => `false`

Reply:
I am dense. :) This was in your description, it just didn't click the first few times I read it.